src/sentry/api/endpoints/organization_metrics.py

```diff
@@ -42,8 +42,10 @@ def get(self, request: Request, organization, metric_name) -> Response:
         projects = self.get_projects(request, organization)
```

Comment:
if your team owns this endpoint, mind adding an entry to CODEOWNERS? 🙏🏼

Reply:
Sure yeah.. how do I do that?

Reply:
Ah you mean like here -> https://github.com/getsentry/sentry/blob/876a24001e607b14f8d87e5999616dd6a0e3dcb8/.github/CODEOWNERS#L25-L36

Reply:
yeah so it'd be something like 

src/sentry/api/endpoints/organization_metrics.py @getsentry/metrics or whatever team :)

Reply:
sounds good! will do, Thanks

Reply:
https://github.com/getsentry/sentry/pull/32749